diff --git a/sysdrv/source/kernel/arch/arm/boot/dts/rv1106.dtsi b/sysdrv/source/kernel/arch/arm/boot/dts/rv1106.dtsi
index 40f0a1a6c..20961febc 100644
--- a/sysdrv/source/kernel/arch/arm/boot/dts/rv1106.dtsi
+++ b/sysdrv/source/kernel/arch/arm/boot/dts/rv1106.dtsi
@@ -224,7 +224,7 @@ route_rgb: route-rgb {
 
 	fiq_debugger: fiq-debugger {
 		compatible = "rockchip,fiq-debugger";
-		rockchip,serial-id = <2>;
+		rockchip,serial-id = <1>;
 		rockchip,wake-irq = <0>;
 		rockchip,irq-mode-enable = <0>;
 		rockchip,baudrate = <115200>;	/* Only 115200 and 1500000 */
@@ -992,7 +992,8 @@ uart1: serial@ff4b0000 {
 		clocks = <&cru SCLK_UART1>, <&cru PCLK_UART1>;
 		clock-names = "baudclk", "apb_pclk";
 		pinctrl-names = "default";
-		pinctrl-0 = <&uart1m0_xfer &uart1m0_ctsn &uart1m0_rtsn>;
+		// pinctrl-0 = <&uart1m0_xfer &uart1m0_ctsn &uart1m0_rtsn>;
+		pinctrl-0 = <&uart1m1_xfer>;
 		status = "disabled";
 	};
 
diff --git a/sysdrv/source/uboot/rkbin/bin/rv11/rv1106_ddr_924MHz_v1.15.bin b/sysdrv/source/uboot/rkbin/bin/rv11/rv1106_ddr_924MHz_v1.15.bin
index 561ee6824..6fe3feec3 100644
Binary files a/sysdrv/source/uboot/rkbin/bin/rv11/rv1106_ddr_924MHz_v1.15.bin and b/sysdrv/source/uboot/rkbin/bin/rv11/rv1106_ddr_924MHz_v1.15.bin differ
diff --git a/sysdrv/source/uboot/rkbin/tools/ddrbin_param.txt b/sysdrv/source/uboot/rkbin/tools/ddrbin_param.txt
index 0dfdd3183..1df9b3e19 100644
--- a/sysdrv/source/uboot/rkbin/tools/ddrbin_param.txt
+++ b/sysdrv/source/uboot/rkbin/tools/ddrbin_param.txt
@@ -9,9 +9,9 @@ lp4_freq=
 lp4x_freq=
 lp5_freq=
 
-uart id=
-uart iomux=
-uart baudrate=
+uart id=1
+uart iomux=1
+uart baudrate=115200
 
 sr_idle=
 pd_idle=
diff --git a/sysdrv/source/uboot/u-boot/arch/arm/dts/rv1106-u-boot.dtsi b/sysdrv/source/uboot/u-boot/arch/arm/dts/rv1106-u-boot.dtsi
index 8821862db..a87b1c091 100644
--- a/sysdrv/source/uboot/u-boot/arch/arm/dts/rv1106-u-boot.dtsi
+++ b/sysdrv/source/uboot/u-boot/arch/arm/dts/rv1106-u-boot.dtsi
@@ -11,7 +11,7 @@
 	};
 
 	chosen {
-		stdout-path = &uart2;
+		stdout-path = &uart1;
 		u-boot,spl-boot-order = &sdmmc, &spi_nor, &spi_nand, &emmc;
 	};
 
diff --git a/sysdrv/source/uboot/u-boot/arch/arm/dts/rv1106.dtsi b/sysdrv/source/uboot/u-boot/arch/arm/dts/rv1106.dtsi
index 7a88bb13e..6b647bb14 100644
--- a/sysdrv/source/uboot/u-boot/arch/arm/dts/rv1106.dtsi
+++ b/sysdrv/source/uboot/u-boot/arch/arm/dts/rv1106.dtsi
@@ -102,7 +102,7 @@
 
 	fiq_debugger: fiq-debugger {
 		compatible = "rockchip,fiq-debugger";
-		rockchip,serial-id = <2>;
+		rockchip,serial-id = <1>;
 		rockchip,wake-irq = <0>;
 		rockchip,irq-mode-enable = <0>;
 		rockchip,baudrate = <1500000>;	/* Only 115200 and 1500000 */
